src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -46,5 +46,20 @@ public async Task<IActionResult> GetBuildStatus(string account, string project,
                         Convert.ToBase64String(Encoding.UTF8.GetBytes(":" + token)));
                 });
         }
+
+        [HttpGet("getFile/{account}/{project}/{repo}/{*filePath}")]
```

Comment:
I'm inexperienced in this area, but I'm not a huge fan of the GET API named "getFile".  Maybe "File" would be preferred, but that doesn't feel quite right either.  @ChadNedzlek @alexperovich ?

Reply:
It's really weird for us to be doing an end run around Azure DevOps security in this why.  We really shouldn't be doing this.  Why not send them to the URL we are about tor return to them?